### ISSUE TITLE
python-uinput2 (fork)

### ISSUE THREAD

**Issue by gitanat** [tuomasjjrasanen/python-uinput#49]
I'd like to announce the creation of **[python-uinput2](https://github.com/pyinput/python-uinput2)**

This is a continuation of the great work done by Tuomas and others, with the following goals:

 - API stability and retro-compatibility
 - Modern python support (3.12+)
 - Keeping support for old python releases until it becomes a significant burden (2.7 is currently supported)

## Why a Fork?

We have decided to fork the original project due to the accumulation of issues, contributions and PRs.

In Tuomas' words: _tempus edax rerum_

## Contributions

Most of the outstanding issues / PRs for python-uinput (1) have already been integrated. We welcome bug reports, contributions and new maintainers.

## Compatibility

Existing code will keep working with no changes (including import statements).
Note that the package name changed to `python-uinput2` (available through pip / pypi).

## Gratitude

I'd like to express my great gratitude for all the work that Tuomas and other contributors put into the software. The fact that it simply keeps working after 8 years with no updates is a testament to how well written it is.

**Comment by gitanat** [comment]
@FeGeyer @hosiet I understand you've helped maintain packaging and patches to the original codebase. This might reduce the maintenance burden (`0001-Migrate-from-distutils-to-setuptools.patch` can be safely deleted). Please feel free to open issues if anything can be improved on the repo side.

**Comment by gitanat** [comment]
@tuomasjjrasanen Obviously you'll always be welcome if you want to be a maintainer. Or if you prefer to continue on this repo please just let us know

**Comment by tuomasjjrasanen** [comment]
@goncalopp Original author here, I'm alive! :)

For starters, I'm happy and grateful to see you (and others) to take part in this project and step up when I have, quite obviously, been neglecting my duty to maintain the project. The story is quite usual, other (good) things have been filling my life and the amount of energy is not unlimited. This project started like so many other open source projects; I had an itch I wanted to scratch and I'm astonished to see that so many others have had the same itch. After all, user space input control is quite niche problem space.

I think this is good time to let it go (should have done it probably years) and hand over the maintainership to others.

And, moving the maintainership to more generic entity, github.com/pyinput in this case, is the way to go.

Now, there are some questions that come to my mind:
- Do you think that forking to a new name is the best way to go, or is there a way to keep the more simple python-uinput alive and not actually affecting users at all?
- If you see, that python-uinput should still be the name, I'd lke to hand over pypi.org/python-uinput as well

**Comment by gitanat** [comment]
@tuomasjjrasanen It's good to hear from you! Being busy, as you say, is a sign that other good things have come - I've also had less time for github over the years. 
This might, as you say, be a fairly niche problem - I guess linux is a bit of a tinkerer magnet! "I wonder if this is possible" is definitely how I found this project in the first place.

Answering your questions:

- I definitely think keeping the same name would be better for everyone if you're onboard with that decision
- Let's coordinate on pypi packaging on #50 

I've added you to github/pyinput in the meantime

**Comment by tuomasjjrasanen** [comment]
Yep, keeping the same name is the best solution for everyone. Basically this is just a transfer of the project management from me to `pyinput`. That said, if you haven't already done so, please change all links and other pointers pointing to my personal web resources etc.: github.com/pyinput is the new manager of this this project, github.com/pyinput/python-uinput is the new home (or are you going to use python-uinput2?, I'd go for python-uinput, but it's your call).

Please also update  the Python package metadata on the next release to match these changes; no need to keep me as the sole author in the metadata, in fact, I think it would be best to declare `pyinput` org as the author. And `author_email` should probably be removed, at least it should not be mine.

**Comment by gitanat** [comment]
We've merged the projects back. The new home for python-uinput is 

https://github.com/pyinput/python-uinput

**Comment by gitanat** [comment]
@tuomasjjrasanen I've changed the metadata, but unfortunately I had released 1.0.0 on pypi before doing so. We can either wait for 1.0.1 or you can [delete the release](https://stackoverflow.com/questions/53070421/remove-package-version-from-pypi) and I'll upload again

**Comment by hosiet** [comment]
When making a future release on pypi, please also tag the release in the Git repository. That would greatly help downstreams that are tracking software releases. Thanks!

**Comment by gitanat** [comment]
Thanks for the reminder @hosiet ! I wasn't sure if we would re-do 1.0.0, but if downstreams are tracking pypi releases I think it'll just be easier on everyone if we immediately release 1.0.1 with the setup metadata changes instead of trying to backfix things

**Comment by gitanat** [comment]
I've released 1.0.1 on pypi with metadata changes and pushed tags
